extensions/big-data-cluster/src/wizards/create-cluster/resourceStrings.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as nls from 'vscode-nls';
+
+const localize = nls.loadMessageBundle();
+
+export const WizardTitle = localize('bdc-create.wizardTitle', 'Create a 2019 Big Data cluster');
```

Comment:
`SQL Server big data cluster`. Let's skip year (when we have SQL 2021 or whatever it'll be same wizard?) and use official product name